markdown/footer.md

```diff
@@ -11,6 +11,7 @@
 
 **[`^        back to top        ^`](#awesome-selfhosted)**
 
+- Alternative frontends/portals to discover/filter awesome-selfhosted apps: [AweXplor](https://awexplor.github.io/aggregated/selfhosted) | [awweso.me](https://awweso.me/) | [awesome-web.theravenhub](https://awesome-web.theravenhub.com/browse.html)
```

Comment:
```suggestion
- Alternative frontends/portals to discover/filter awesome-selfhosted apps: [awweso.me](https://awweso.me/) | [awesome-web.theravenhub](https://awesome-web.theravenhub.com/browse.html)
```

Extremely out-of-date data, violates CC-BY-SA 4.0 license (no link to source or mention of the license)